lldb/source/Commands/CommandObjectBreakpoint.cpp

```diff
@@ -153,6 +153,14 @@ class lldb_private::BreakpointOptionGroup : public OptionGroup {
         m_bp_opts.GetThreadSpec()->SetIndex(thread_index);
       }
     } break;
+    case 'Y': {
+      LanguageType language = Language::GetLanguageTypeFromString(option_arg);
```

Comment:
Since this is supposed to be a language that we can use to run expressions, you should check the language you got from the option against `Language::GetLanguagesSupportingTypeSystemsForExpressions()`.  It would be wrong to allow people to set a language for a condition expression to one for which we can't evaluate expressions.